.github/workflows/milestone.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - name: Get milestone from pom.xml
         run: |
           .github/bin/retry ./mvnw -v
+          ./mvnw help:evaluate -Dexpression=project.version -q -DforceStdout | cut -d- -f1
```

Comment:
This is not necessary, you can check env vars in the next step when you expand the `Run ...` section:
![image](https://user-images.githubusercontent.com/795177/234633178-05d04dad-27cb-4498-81ff-8de0d19cad6c.png)

Reply:
I want to debug those failures: https://github.com/trinodb/trino/actions/runs/4748403112/jobs/8434562002

Reply:
can you debug without merging this PR, by modifying 
```
on:
  push:
    branches: [master]
```
?

Reply:
It's useful to have such output, doesn't cost anything and helps debug issues when they happen instead of trying to reproduce them.